tests/integration/test_amendment_chain.py

```diff
@@ -294,12 +308,21 @@ def test_negative_filing_chain(self):
         assert len(results) == 1
 
         result = results[0]
```

Comment:
@hcaofec can you remove this print statement?

Reply:
Thanks for catching this.